page/about.html

```diff
@@ -0,0 +1,234 @@
+                    About 
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#contact-us"  class="mob-menu-item" >
+                    Contact Us
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#logo"  class="mob-menu-item" id="sponsor">
+                    Current Logo 
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#past-events"  class="mob-menu-item" id="join">
+                    Past summits 
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+            <li class="mob-side-nav">
+                <a href="#partner"  class="mob-menu-item" id="news">
+                    Partner 
+                    <img src="../assets/icons/chevron_right.png" alt="">
+                </a>
+            </li>
+        </ul>
+    </section>
+
+    <section class="about-intro-container">
+        <header>
+            <ul class="most-top">
+                <li>
+                    <a href="#" class="icon">
+                        <i class="fab fa-facebook fa-fw"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="icon">
+                        <i class="fab fa-twitter fa-fw"></i>
+                    </a>
+                </li>
+                <li class="icon">English</li>
+                <li>
+                    <a href="#" class="icon">My Page</a>
+                </li>
+                <li>
+                    <a href="#" class="icon">Logout</a>
+                </li>
+            </ul>
+            <span class="mobile-menu mob-side-nav">
+                <img src="../assets/icons/menuIcon.svg" alt="Humburgur Menu">
+            </span>
+    
+        </header>
+
+        <section class="home-intro">
+
+            <div class="page-intro">
+                <h1 id="title-desc">International Technology Talk Summit 2022</h1>
+                <div class="opening-desc">
+                    <p>The Global summit brings together the community of experts, academics adn activists who comprise the Creative Commmons affilate network in a different country every tow this year, Wehope to expand our inivitation list including organizationa ans individiduals.</p>
+                </div>
+            </div>
+            
+        </section>
+
+    </section>
+
+    <section class="contact-us" id="contact-us">
+        <p>Please contact us per Email for any further questions about CC Global Summit</p>
+        <a>bush7840@yahoo.com</a>
+    </section>
+
+    <section class="logo" id="logo">
+        <h1>Creative commons Global Summit 2015 Logo</h1>
```

Comment:
- [ ] As earlier highlighted by the last reviewer. For semantic HTML, there shouldn't be more than one h1 tag per page. The h1 tag should be used for the main header of a page. Kindly replace all other h1 tags on this page with other hx tags.

Reply:
Noted and solved